Updated compile subtype part (#155)
AlexsLemonade/OpenPBTA-manuscript: content/03.methods.md

The compilation was executed by the following steps:

Firstly, `broad_histology`, `short_histology`, and `integrated_diagnosis` columns in the result files from the above subtyping modules (i.e., `CRANIO_molecular_subtype.tsv`, `EWS_results.tsv`, `EPN_all_data_withsubgroup.tsv`, `HGG_molecular_subtype.tsv` ,`lgat_subtyping.tsv`, `MB_molecular_subtype.tsv`, `embryonal_tumor_molecular_subtypes.tsv`, and `neurocytoma_subtyping.tsv`) were updated based on the molecular subtype of the tumor. 
Detailed information about the updating procedure were included in the analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/01-compile-subtyping-results.Rmd).
Notes were also added to indicated that the changes in `broad_histology`, `short_histology` and `integrated_diagnosis` were from OpenPBTA subtyping modules.

Subsequently, `broad_histology`, `short_histology` and `harmonized_diagnosis` columns of tumors with particular pathology diagnosis free text were updated as specified in the following table:

> pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | harmonized_diagnosis

Reviewer: ```suggestion

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | harmonized_diagnosis
```

Other | NA | contains `xanthogranuloma` or `jxg` | Histiocytic tumor | JXG | Juvenile xanthogranuloma
Meningioma | NA | contains `atypical` | Meningioma | Meningioma | Atypical meningioma
Meningioma | NA | contains `anaplastic` | Meningioma | Meningioma | Anaplastic (malignant) meningioma
Meningioma | NA | contains `clear cell meningioma` | Meningioma | Meningioma | Clear cell meningioma
Meningioma | NA | contains `meningothelial` | Meningioma | Meningioma | Meningothelial meningioma
Meningioma | NA | does not contain `atypical`, `anaplastic`, `clear cell`, or `meningothelial` | Meningioma | Meningioma | Meningioma
Choroid plexus papilloma | NA | contains `atypical` | Choroid plexus tumor | Choroid plexus tumor | Atypical choroid plexus papilloma
Craniopharyngioma | CRANIO | contains `adamantinomatous` | Tumors of sellar region | Craniopharyngioma | Adamantinomatous craniopharyngioma 

Similarly, `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` columns of tumors with following pathology diagnosis free text were updated as specified in the table below:

> pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | integrated_diagnosis

Reviewer: ```suggestion

pathology_diagnosis | subtyping module | pathology_free_text_diagnosis | broad_histology | short_histology | integrated_diagnosis
```

Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `diffuse leptomeningeal glioneuronal tumor` | Neuronal and mixed neuronal-glial tumor | GNT | Diffuse leptomeningeal glioneuronal tumor, <subtype> | Diffuse leptomeningeal glioneuronal tumor
Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `glioneuronal` | Neuronal and mixed neuronal-glial tumor | GNT | Glial-neuronal tumor NOS, <subtype> | Glial-neuronal tumor NOS
Low-grade glioma/astrocytoma (WHO grade I/II) | NA, remove from LGAT module | contains `rosette forming glioneuronal tumor` | Neuronal and mixed neuronal-glial tumor | GNT | Rosette-forming glioneuronal tumor, <subtype> | Rosette-forming glioneuronal tumor

Notes were also added to indicated that the changes in `broad_histology`, `short_histology`, `integrated_diagnosis` and `harmonized_diagnosis` were from pathology diagnosis free text.

> For PNOC003 samples, `HGG` or `DMG` were assigned through HGG molecular subtyping module. 
> This may or may not be consistent with the molecular subtype assigned in clinical review. 
> For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated to reflect clinical review.
> Notes were also added to indicate updates were from clinical review. 

Reviewer 2: ```suggestion
For samples with subtype discrepancies, `molecular_subtype` and `integrated_diagnosis` were updated following pathology or clinical review.
```

> Detailed information can be found in the analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd).

Reviewer 2: ```suggestion
Detailed information can be found in the analysis notebooks for [clinical](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/03-incorporate-clinical-feedback.Rmd) and [pathology](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd) feedback.
```

> Lastly, some samples were manually inspected by pathologists in CHOP and if discrepancies occur, `broad_histology`, `short_histology`, `integrated_diagnosis` and `molecular_subtype` were updated based on pathologist review.

Reviewer 2: ```suggestion
```

> Detailed information can be found in analysis [notebook](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/molecular-subtyping-pathology/04-incorporate-pathology-feedback.Rmd).

Reviewer 2: ```suggestion
```
